package.json

```diff
@@ -15,36 +22,71 @@
     "@types/node": "^16.11.39",
     "@types/react": "^18.0.12",
     "@types/react-dom": "^18.0.5",
+    "@types/react-redux": "^7.1.25",
+    "axios": "^1.3.5",
```

Comment:
If we are adopting a UI library (MUI, or Prime) it would be preferable to using their techniques instead of bringing in Bootstrap for a few isolated instances. (I think I found it used only in `vendor.scss`)